apisix/stream/xrpc/runner.lua

```diff
@@ -90,7 +90,7 @@ local function filter_logger(ctx, logger)
     end
 
     -- key and version are divided by per-logger level
-    local key = "xrpc-logger" .. ctx.conf_id
+    local key = tostring(ctx.conf_id)
```

Comment:
We can save `tostring` here as `ctx.conf_id` is already a string?

Reply:
update